### ISSUE TITLE
Media in private groups or channels appear blank.

### ISSUE THREAD

**Issue by nasyxx** [overtake/TelegramSwift#1151]
All media (including pictures,videos) in private groups/channels are shown blank.

<img width="264" alt="Screenshot 2024-07-31 at 12 28 54 PM" src="https://github.com/user-attachments/assets/a00307ff-fa63-48ee-a125-c153a691b07e">

My macOS is Sequoia 15.1 Beta (24B5009l).  Telegram: 10.15.3.266648 Stable

**Comment by zweeltow** [comment]
Still reproduced at latest version? I've never encountered this problem.

**Comment by nasyxx** [comment]
> Still reproduced at latest version? I've never encountered this problem.

I just found that `10.15.3.266641 Beta` fixed the problem.

**Comment by nasyxx** [comment]
> > Still reproduced at latest version? I've never encountered this problem.
> 
> I just found that `10.15.3.266641 Beta` fixed the problem.

No, it's the same version and it happened again.  not sure why.

**Comment by nasyxx** [comment]
> Still reproduced at latest version? I've never encountered this problem.

Is there any way I can debug this problem?

**Comment by nasyxx** [comment]
> Still reproduced at latest version? I've never encountered this problem.

Hi, I found out why this happened.

When I extend my screen using iPad, the media will become unavailable.
